test/flows/marriage_abroad_flow_test.rb

```diff
@@ -0,0 +1,436 @@
+    end
+
+    should "render question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next_node of partner_opposite_or_same_sex? for a known country" do
+        assert_next_node :outcome_marriage_abroad_in_country, for_response: "opposite_sex"
+      end
+
+      should "raise an error for an unknown country" do
+        stub_worldwide_api(%w[narnia])
+        add_responses country_of_ceremony?: "narnia"
+        assert_raises(SmartAnswer::Question::Base::NextNodeUndefined) do
+          add_response "opposite_sex"
+          @test_flow.state
+        end
+      end
+    end
+  end
+
+  context "outcome :outcome_marriage_abroad_in_country" do
+    setup { testing_node :outcome_marriage_abroad_in_country }
+
+    MARRIAGE_OR_PACS_COUNTRIES.each do |country|
+      context "marriage or pacs country: #{country}" do
+        setup do
+          stub_worldwide_api([country])
+          add_responses country_of_ceremony?: country
+        end
+
+        should "render a marriage outcome" do
+          add_responses marriage_or_pacs?: "marriage"
+          assert_rendered_outcome
+        end
+
+        should "render a PACS outcome" do
+          add_responses marriage_or_pacs?: "pacs"
+          assert_rendered_outcome
+        end
+      end
+    end
+
+    CEREMONY_LOCATION_COUNTRIES.each do |country|
+      context "ceremony location country: #{country}" do
+        setup do
+          stub_worldwide_api([country])
+          add_responses country_of_ceremony?: country
+        end
+
+        should "render an outcome where residency is in the ceremony country" do
+          add_responses legal_residency?: "ceremony_country"
+          assert_rendered_outcome
+        end
+
+        should "render an outcome where residency is in a different country" do
+          add_responses legal_residency?: "third_country"
+          assert_rendered_outcome
+        end
+
+        should "render an outcome where residency is in the UK" do
+          add_responses legal_residency?: "uk"
+          assert_rendered_outcome
+        end
+      end
+    end
+
+    ONE_QUESTION_COUNTRIES.each do |country|
+      context "one question country" do
+        setup do
+          stub_worldwide_api([country])
+          add_responses country_of_ceremony?: country
+        end
+
+        should "render an outcome" do
+          assert_rendered_outcome
+        end
+      end
+    end
+
+    TWO_QUESTION_COUNTRIES.each do |country|
+      context "two question country: #{country}" do
+        setup do
+          stub_worldwide_api([country])
+          add_responses country_of_ceremony?: country
+        end
+
+        should "render an opposite sex outcome" do
+          add_responses partner_opposite_or_same_sex?: "opposite_sex"
+          assert_rendered_outcome
+        end
+
+        should "render a same sex outcome" do
+          add_responses partner_opposite_or_same_sex?: "opposite_sex"
```

Comment:
should be `partner_opposite_or_same_sex?: "same_sex"`

Reply:
Ooh good catch 👍